protocol/persistence.go

```diff
@@ -561,21 +561,21 @@ func (db sqlitePersistence) Contacts() ([]*Contact, error) {
 	for rows.Next() {
 
 		var (
-			contact             Contact
-			nickname            sql.NullString
-			contactRequestState sql.NullInt64
-			contactRequestClock sql.NullInt64
-			displayName         sql.NullString
-			imageType           sql.NullString
-			ensName             sql.NullString
-			ensVerified         sql.NullBool
-			added               sql.NullBool
-			blocked             sql.NullBool
-			removed             sql.NullBool
-			hasAddedUs          sql.NullBool
-			lastUpdatedLocally  sql.NullInt64
-			identityImageClock  sql.NullInt64
-			imagePayload        []byte
+			contact                   Contact
+			nickname                  sql.NullString
+			contactRequestLocalState  sql.NullInt64
+			contactRequestLocalClock  sql.NullInt64
+			contactRequestRemoteState sql.NullInt64
+			contactRequestRemoteClock sql.NullInt64
+			displayName               sql.NullString
+			imageType                 sql.NullString
+			ensName                   sql.NullString
+			ensVerified               sql.NullBool
+			blocked                   sql.NullBool
```

Comment:
For bools, I think it's probably ok to not have null fields, I think sqlite handles this values fine and they will always be defaulted to some non-null valu.

For strings for example, they would be `nil` in the database on migrations when adding a column (these are all columns that have been added after the initial table was created), and we don't want to set those fields on a migration, since it might take a bit longer to migrate on some potentially large collections (as `contacts` might be), which leads to longer login time after an upgrade.

Therefore if we are not writing or defaulting these fields, we use nullvalues for those.